.github/CONTRIBUTING.md

```diff
@@ -64,6 +64,9 @@ echo "VELA_SOURCE_CLIENT=<Github Client ID>" >> secrets.env
 
 # Add Github Client Secret to local secrets file for `docker-compose`
 echo "VELA_SOURCE_SECRET=<Github Client Secret>" >> secrets.env
+
+# Disable webhook validation
+echo "VELA_DISABLE_WEBHOOK_VALIDATION=true" >> secrets.env
```

Comment:
Thoughts on adding this environment variable directly to the Docker compose file?

I ask because I wonder if there is a scenario where we wouldn't want that set in the compose stack. I believe it is primarily used for development on local workstations so thinking this could be set there 👍